utest/test/keywords/test_keyword_arguments_browsermanagement.py

```diff
@@ -31,3 +31,8 @@ def test_open_browser(self):
         alias = self.brorser.open_browser(url, alias='None',
                                           remote_url=remote_url)
         self.assertEqual(alias, None)
+
+        when(self.brorser)._make_driver('firefox', None,
+                                        None, False, None, None).thenReturn(browser)
+        alias = self.brorser.open_browser()
+        self.assertEqual(alias, None)
```

Comment:
Instead of writing this assert, something like this could be better:
```
from mockito import verify, ANY

verify(brorser, times=0).get(ANY)
```

Written without testing the code, so errors might be present, Please let me know, if you want me to try out provide more accurate example.

Reply:
Just for clarity's sake - you want me to 1) change the assertEqual into the mokito verify and 2) add another test?

Reply:
Okidoke - that should do it :)